app/js/lib/view_handler/element_slide.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+
+	/**
+	 * View Handler for showing and hiding elements if the update value matches a regular expression
+	 * @class
+	 */
+	ElementSlider = {
```

Comment:
At first glance I was expecting this to be some kind of control element, while it actually is an encapsulation of animation stuff. Perhaps something like ElementSlideAnimator?